README.md

```diff
@@ -6,4 +6,6 @@ DAL adapter for Aver PTZ cameras
 
 **Build project with all tests: ``` mvn clean install ```**
 
-**Build project with only development tests: ``` mvn clean install -P DevelopmentTest```**
\ No newline at end of file
+**Build project with only development tests: ``` mvn clean install -P DevelopmentTest```**
```

Comment:
if we change the name to real device and mock, we should update the readme file as well

Reply:
ok thank you, I will update it